fall23/deepface/detectfaces.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.    fr.detect_faces_from_images()
</pre><p>It would be good to add some observations on the results. Similar to what was discussed this week (meeting on 12/04/2023). Bar graphs with number of detections/matches would help when comparing various models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-atfal-mafkoda-missing-children/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Sure kabilan will add them.